lib/active_model/serializer/adapter/cached_serializer.rb

```diff
@@ -24,7 +24,7 @@ def cached?
         end
 
         def fragment_cached?
-          @klass._cache_only && !@klass._cache_except || !@klass._cache_only && @klass._cache_except
+          @klass._cache && (@klass._cache_only && !@klass._cache_except || !@klass._cache_only && @klass._cache_except)
```

Comment:
this can be it's own PR. would you mind? https://github.com/CheckMateIO/active_model_serializers/commit/e9e2b5712df0eda12399bac31a5d126b8b040082